client/lib/site/index.js

```diff
@@ -78,7 +86,39 @@ Site.prototype.set = function( attributes ) {
 	}
 
 	return changed;
+};
+
+/**
+ * Replaces all site's attributes. Will remove current attributes if they are not present in the
+ * new ones. Use Site.prototype.set if you want to update one or few attributes instead.
+ *
+ * @param {object} attributes - New attributes for a site.
+ * @returns {boolean} - Returns true if attributes changed, otherwise false.
+ */
+Site.prototype.replace = function( attributes ) {
+	var changed = false,
```

Comment:
We [prefer](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md#variable-declarations) `const`. It's OK if the style is not consistent with the rest of the file.